bigbluebutton-client/src/org/bigbluebutton/modules/whiteboard/views/WhiteboardCanvas.as

```diff
@@ -335,11 +345,6 @@ package org.bigbluebutton.modules.whiteboard.views {
 			currentWhiteboardId = wbId;
 			canvasDisplayModel.changeWhiteboard(wbId);
 			whiteboardToolbar.whiteboardIdSet();
-			
-			var multiUser:Boolean = whiteboardModel.getMultiUser(wbId);

```

Comment:
multiuser is per-whiteboard so you need to fetch the value every time you switch whiteboards.